core/trino-main/src/test/java/io/trino/sql/planner/TestPartialTranslator.java

```diff
@@ -67,15 +75,24 @@ public void testPartialTranslator()
         Expression rowSymbolReference = new SymbolReference("row_symbol_1");
         Expression dereferenceExpression1 = new SubscriptExpression(rowSymbolReference, new LongLiteral("1"));
         Expression dereferenceExpression2 = new SubscriptExpression(rowSymbolReference, new LongLiteral("2"));
+        Expression dereferenceExpression3 = new SubscriptExpression(rowSymbolReference, new LongLiteral("3"));
         Expression stringLiteral = new StringLiteral("abcd");
         Expression symbolReference1 = new SymbolReference("double_symbol_1");
+        SymbolReference timestamp3SymbolReference = new SymbolReference("timestamp3_symbol_1");
 
         assertFullTranslation(symbolReference1);
         assertFullTranslation(dereferenceExpression1);
         assertFullTranslation(stringLiteral);
 
-        Expression binaryExpression = new ArithmeticBinaryExpression(ADD, symbolReference1, dereferenceExpression1);
-        assertPartialTranslation(binaryExpression, ImmutableList.of(symbolReference1, dereferenceExpression1));
+        assertPartialTranslation(
+                new ArithmeticBinaryExpression(ADD, symbolReference1, dereferenceExpression1),
+                List.of(symbolReference1, dereferenceExpression1));
+
+        assertPartialTranslation(
+                new CoalesceExpression(
```

Comment:
i replaced `+` with `coalesce` as an example of an expression that cannot be translated.
This is OK for now, but we need a better testing strategy, see eg https://github.com/trinodb/trino/pull/11535 
cc @ebyhr